src/parser/core/Events.ts

```diff
@@ -216,6 +216,9 @@ export interface EndChannelEvent extends Event<EventType.EndChannel> {
   start: number;
   duration: number;
   beginChannel: BeginChannelEvent;
+  trigger?: {
+    timestamp: number;
+  };
 }
```

Comment:
where are these event changes used?

Reply:
If this is only one field then why are we making a whole new json for it?

Reply:
They are used in filtered active time.

There are more under trigger, I just only needed the one and I didn't add the rest